### ISSUE TITLE
Cannot access class 'org.w3c.dom.Storage'

### ISSUE THREAD

**Issue by GuilhE** [russhwolf/multiplatform-settings#209]
This error was throwing in this configuration, for `StorageSettings`:
```
actual class PlatformDependencies actual constructor() {
    actual fun getSettings(): Settings = StorageSettings()
}
```
The issue was resolved by adding the [kotlinx-browser](https://github.com/Kotlin/kotlinx-browser) dependency to the `wasmJsMain` target in the `shared` module. Is this something users need to handle in their projects, or could `multiplatform-settings` address it internally? It might also be worth mentioning in the README.md.

Thanks.

**Comment by russhwolf** [comment]
The sample app doesn't include kotlinx-browser and builds/runs fine as far as I'm aware. Can you link a project that reproduces the issue?

**Comment by russhwolf** [comment]
Ok I see. The APIs were removed from the wasm stdlib in 2.1.0-Beta2 which I hadn't gotten around to testing against. I'll add the dependency in the next release.

**Comment by russhwolf** [comment]
Now released in 1.3.0

**Comment by gleb-skobinsky** [comment]
> Now released in 1.3.0

FYI, I still encountered this compile-time error with version 1.3.0. This might be because of a custom source set layout I use. Adding the kotlinx-browser dependency helped.
